src/main.ts

```diff
@@ -70,7 +70,12 @@ async function start () {
 
   // configure Keycloak Service
   console.info('Starting Keycloak service...') // eslint-disable-line no-console
-  await KeycloakService.setKeycloakConfigUrl(store.getters.getKeycloakConfigPath)
+  const keycloakConfig: any = {
+    url: 'https://dev.loginproxy.gov.bc.ca/auth',
+    realm: 'bcregistry',
+    clientId: 'entity-web'
+  }
+  await KeycloakService.setKeycloakConfigUrl(keycloakConfig)
```

Comment:
I imported this change from a change Patrick recently made. It will go away after a rebase.

Reply:
It's gone now (from this PR; obviously it's still in the main branch).